src/SFA.DAS.EmployerCommitmentsV2.Web/Startup/ConfigurationStartup.cs

```diff
@@ -9,7 +9,10 @@ public static class ConfigurationStartup
         public static IWebHostBuilder ConfigureDasAppConfiguration(this IWebHostBuilder hostBuilder)
         {
             return hostBuilder.ConfigureAppConfiguration(c => c
-                .AddAzureTableStorage(ConfigurationKeys.EmployerCommitmentsV2));
+                .AddAzureTableStorage(ConfigurationKeys.EmployerCommitmentsV2)
+                .AddAzureTableStorage(ConfigurationKeys.Encoding)
+                .AddAzureTableStorage(ConfigurationKeys.EmployerAccountsApiClient)
```

Comment:
Are the `EmployerAccountsApiClient` & `EmployerAccountsReadStore` configs needed?

Reply:
removed